### PR TITLE
Devop: update network selection

### PR DESCRIPTION
**Pull request by Abyscuit**
### Devop

* \[x] Added entry to ./changelog
* \[x] Add PR label

### CODE REVIEW

File: src/core/router/routes-offline-wallet.js [hunk 1]
```diff
@@ -13,14 +13,14 @@ export default {
...
     },
-    {
-      path: ROUTES_WALLET.NETWORK.PATH,
-      name: ROUTES_WALLET.NETWORK.NAME,
-      component: () => import('@/modules/network/ModuleNetwork'),
-      meta: {
-        noAuth: false
-      }
-    },
+    // {
```
Comment (gamalielhere) [inline src/core/router/routes-offline-wallet.js:16]: Remove commented code.

File: src/core/router/routes-wallet.js [hunk 2]
```diff
@@ -67,14 +67,14 @@ export default {
...
     },
-    {
-      path: ROUTES_WALLET.NETWORK.PATH,
-      name: ROUTES_WALLET.NETWORK.NAME,
-      component: () => import('@/modules/network/ModuleNetwork'),
-      meta: {
-        noAuth: false
-      }
-    },
+    // {
```
Comment (gamalielhere) [inline src/core/router/routes-wallet.js:70]: Here too.

File: src/views/layouts-wallet/TheSwapLayout.vue [hunk 3]
```diff
@@ -35,13 +35,7 @@ export default {
   },
   mixins: [handlerAnalytics],
   beforeRouteLeave(to, from, next) {
-    if (to.name === ROUTES_WALLET.NETWORK.NAME) {
-      this.trackSwap('switchingNetworkOnSwap');
```
Comment (gamalielhere) [inline src/views/layouts-wallet/TheSwapLayout.vue:39]: Move this to check when the popup is being opened in the swap page.

**Review by gamalielhere — CHANGES_REQUESTED** [review]

File: src/views/layouts-wallet/TheSwapLayout.vue [hunk 4]
```diff
@@ -35,13 +35,7 @@ export default {
...
-    }
-    if (
-      to.name !== ROUTES_WALLET.NETWORK.NAME &&
-      to.name !== ROUTES_WALLET.SWAP.NAME
-    ) {
+    if (to.name !== ROUTES_WALLET.SWAP.NAME) {
```
Comment (gamalielhere) [inline src/views/layouts-wallet/TheSwapLayout.vue:38]: Route doesn't change anymore.  Watch the "isOpenNetworkOverlay" instead, check if current route is swap, then fire event.

File: src/modules/network/ModuleNetwork.vue [hunk 5]
```diff
@@ -83,19 +85,18 @@ export default {
     }
   },
   mounted() {
-    this.openNetworkOverlayOnLoad();
+    // this.openNetworkOverlayOnLoad();
```
Comment (gamalielhere) [inline src/modules/network/ModuleNetwork.vue:88]: Remove commented code.

**Review by gamalielhere — CHANGES_REQUESTED** [review]

**Review by gamalielhere — APPROVED** [review]

**Review by jmjles — APPROVED** [review]

**Pull request merged**